state-chain/pallets/cf-pools/src/lib.rs

```diff
@@ -1245,12 +1252,18 @@ pub struct PoolOrderbook {
 }
 
 #[derive(Clone, Debug, Encode, Decode, TypeInfo, PartialEq, Eq, Deserialize, Serialize)]
-pub struct PoolPrice {
+pub struct PoolPriceV1 {
 	pub price: Price,
 	pub sqrt_price: SqrtPriceQ64F96,
 	pub tick: Tick,
 }
 
+#[derive(Serialize, Deserialize, Clone, Encode, Decode, TypeInfo, PartialEq, Eq, Debug)]
+pub struct PoolPriceV2 {
+	pub sell: Option<SqrtPriceQ64F96>,
+	pub buy: Option<SqrtPriceQ64F96>,
```

Comment:
Didn't one of the LPs have a preference for ticks? What's the reason for removing the additional information?

Reply:
Had a very long discussion with them, they didn't understand a bunch of things. The conclusion was actually they only ever wanted the sqrt_price anyway, the others are possibly/likely misleading.